fixtures/tracks/animal/exercises/TRACK_HINTS.md

```diff
@@ -0,0 +1 @@
+This is the content of the track hints file
```

Comment:
this creates an `exercises` dir in `animal`, which means any exercises in `animal` (namely `dog` and `fish`) outside the `exercises` dir will never have any files, because https://github.com/exercism/trackler/blob/master/lib/trackler/implementation.rb#L46

If this should be supported, `Implementation` will need to change to look first for `exercises/$slug` and then fall back to `$slug`, *even if `exercises/` exists*.

If this should not be supported, the `animal` track now uses unsupported features, and could break at any time if a test relies on the integrity of the `animal` track.